aws/service_v2.go

```diff
@@ -131,6 +157,106 @@ func DynamoDbClient(ctx context.Context, d *plugin.QueryData) (*dynamodb.Client,
 	return svc, nil
 }
 
+// Ec2Client returns the service client for AWS Ec2 service
+func Ec2Client(ctx context.Context, d *plugin.QueryData) (*ec2.Client, error) {
+	region := d.KeyColumnQualString(matrixKeyRegion)
+	if region == "" {
+		return nil, fmt.Errorf("region must be passed DynamodbClient Client")
```

Comment:
```suggestion
		return nil, fmt.Errorf("region must be passed for EC2 Client")
```

The reference to `DynamodbClient` also seems incorrect in other new functions (maybe in old functions too).